tests/integration/server.tap.js

```diff
@@ -5,18 +5,20 @@
 
 'use strict'
 
-var tap = require('tap')
-var http = require('http')
-var natives = require('../../')()
-var segs = require('segfault-handler')
+const tap = require('tap')
+const http = require('http')
+const segs = require('segfault-handler')
+const { execSync } = require('child_process')
 
-var RUN_TIME = 5 * 60 * 1000 // 5 minutes
+const RUN_TIME = 5 * 60 * 1000 // 5 minutes
 
 segs.registerHandler('crash.log')
 
 tap.test('server soak test', {timeout: RUN_TIME + 10000}, function(t) {
+  execSync(`node ./lib/pre-build install native_metrics`)
+  const natives = require('../../')()
```

Comment:
Curious the reason for adding the child process install and then delayed include for this test which has no other tweaks?

Reply:
So in the other integration tests I clean out the `build/Release` folder before every run to ensure clean runs. This affected the other tests, mostly this one because it assumed the binary was built from the `npm install` stage. by forcing the install and then including after it fixed these tests

Reply:
ah. prefer not relying on prior state. 👍